config/systems/todi.py

```diff
@@ -76,6 +76,46 @@
                 ],
             'launcher': 'srun',
         },
+        {
+            'name': 'debug',
+            'descr': 'GH200',
+            'scheduler': 'squeue',
```

Comment:
```suggestion
            'scheduler': 'slurm',
```
Using `squeue` circumvents `sacct` issues on `todi-ln001`.